samples/highcharts/demo/polar-advanced/demo.js

```diff
@@ -0,0 +1,479 @@
+            const { chartWidth, chartHeight } = this.chart;
+            return {
+                x: (chartWidth / 2) - (labelWidth / 2),
+                y: (chartHeight / 2) - (labelHeight / 2)
+            };
+        }
+    },
+    colorAxis: [{
+        minColor: hcColor0.brighten(0.05).get('rgba'),
+        maxColor: hcColor5.brighten(0.05).get('rgba'),
+        showInLegend: false,
+        ...weekExtremes
+    }, {
+        minColor: hcColor1.tweenTo(hcColor5, 0.5),
+        maxColor: asHcColor(colors[8]).tweenTo(asHcColor(colors[8]), 0.5),
+        showInLegend: false,
+        ...monthExtremes
+    }
+    ],
+    pane: [{
+        size: '95%',
+        innerSize: '60%',
+        ...paneOpeningAngles,
+        background: {
+            borderColor: asHcColor(colors[4]).get('rgba'),
+            backgroundColor: toggleableGradient,
+            outerRadius: '60%'
+        }
+    }, {
+        size: '55%',
+        innerSize: '45%',
+        ...paneOpeningAngles,
+        background: {
+            borderWidth: 0,
+            backgroundColor: toggleableGradient,
+            outerRadius: '75%'
+        }
+    }, {
+        size: '100%',
+        innerSize: '88%',
+        startAngle: 16.5,
+        endAngle: 343.5,
+        background: {
+            borderWidth: 1,
+            borderColor: asHcColor(colors[4]).get('rgba'),
+            backgroundColor: '#46465C',
+            innerRadius: '55%',
+            outerRadius: '100%'
+        }
+    }],
+    xAxis: [{
+        pane: 0,
+        tickInterval: 1,
+        lineWidth: 0,
+        gridLineWidth: 0,
+        min: 1,
+        max: 26,
+        ...noLabelProp
+    }, {
+        pane: 1,
+        linkedTo: 0,
+        gridLineWidth: 0,
+        lineWidth: 0,
+        plotBands: Array(3).fill(7).map(
+            (weekendOffset, week) => {
+                const
+                    from = weekendOffset * (week + 1),
+                    to = from - 1;
+                return { from, to, color: '#BBBAC5' };
+            }
+        ),
+        ...monthExtremes,
+        ...noLabelProp
+    }, {
+        pane: 2,
+        tickAmount: 4,
+        tickInterval: 0.5,
+        gridLineWidth: 0,
+        lineWidth: 0,
+        ...weekExtremes,
+        ...noLabelProp
+    }],
+    yAxis: [{
+        pane: 0,
+        tickInterval: 8,
+        gridLineWidth: 0.5,
+        gridLineDashStyle: 'longdash',
+        gridLineColor: '#BBBAC5',
+        max: 1800,
+        min: -8,
+        title: null,
+        ...noLabelProp
+    }, {
+        pane: 1,
+        reversed: true,
+        gridLineWidth: 0,
+        tickInterval: 100,
+        min: 0,
+        max: 400,
+        title: null,
+        ...noLabelProp
+    }, {
+        pane: 2,
+        tickInterval: 0.25,
+        gridLineWidth: 0,
+        gridLineColor: hcColor1.brighten(0.05).get('rgba'),
+        min: -3,
+        max: 1,
+        title: null,
+        ...noLabelProp
+    }],
+    legend: {
```

Comment:
When I'm hovering any legend item it applies the default `itemHoverStyle` with black font color, I think we should change it.

Reply:
Done!